Foundation/NSCharacterSet.swift

```diff
@@ -168,6 +168,10 @@ public class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
         return CFCharacterSetCreateCopy(kCFAllocatorSystemDefault, self._cfObject)
     }
     
+    public override func mutableCopy() -> AnyObject {
```

Comment:
I have filed another related issue to this https://bugs.swift.org/browse/SR-310
it is probably the same underlying problem